Dockerfile

```diff
@@ -1,10 +1,9 @@
 FROM registry.fedoraproject.org/fedora-minimal:latest
 
-ARG ISO_URL
 ENV IMAGE '/data/image.iso'
 ENV IPXE_DIR '/data/ipxe'
 ENV BASE_URL http://localhost:8888
-ENV KERNEL_OPTS 'random.trust_cpu=on rd.luks.options=discard ignition.firstboot ignition.platform.id=metal console=tty1 console=ttyS1,115200n8 coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8"'
```

Comment:
how would you override the kernel options (e.g serial console) without it?

Reply:
I just moved these values into the shell script so they're present when running it outside of a container.

Reply:
They can still be overridden with an env var when running the container.